### PR TITLE
Support spark schema as DDL string

### PR DESCRIPTION
**Pull request by RuneLind**
Added schemaType to SchemaWithOptions to specify schema type. Disabled module system to make it work with Scala libs for spark.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -121,6 +121,27 @@
...
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-avro_2.12</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-sql_2.12</artifactId>
+            <version>3.0.0</version>
+            <scope>provided</scope>
```
Comment (bjornandre) [inline pom.xml:133]:
```suggestion
```

Comment (bjornandre) [reply pom.xml:133]: Must be compile scope to be included in target/lib

Comment (RuneLind) [reply pom.xml:133]: Find it a strange this compiled and was able to run tests with this set as provided.

Comment (bjornandre) [reply pom.xml]: Yes, but the docker service failed at startup

**Review by bjornandre — APPROVED** [review]

**Pull request merged**